source/process/software-requirements.rst

```diff
@@ -0,0 +1,38 @@
+=================================
+Mattermost Software Requirements
+=================================
+
+This document provides easy-to-follow guidelines for determining which software versions Mattermost requires. For past discussion on why these guidelines were chosen, see `this conversation <https://pre-release.mattermost.com/core/pl/sb4fq6qhyfbb5xjdp7x3ud146e>`_.
+
+Current software requirements are `documented here <https://docs.mattermost.com/install/requirements.html#software-requirements>`_.
+
+Desktop Apps
+---------------------------------
+
+.. csv-table::
+    :header: "Operating System", "Guideline"
+
+    "Windows", "Supported versions by Microsoft - `reference <https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions>`_"
```

Comment:
Wikipedia is probably not the most official source for this info, is there a page on the windows or apple sites with this info?